tests/client/amo/components/TestFeaturedAddons.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import * as featuredActions from 'amo/actions/featured';
+import {
+  FeaturedAddonsBase,
+  mapStateToProps,
+} from 'amo/components/FeaturedAddons';
+import createStore from 'amo/store';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import { fakeAddon } from 'tests/client/amo/helpers';
+import { getFakeI18nInst, shallowRender } from 'tests/client/helpers';
+
+
+describe('<FeaturedAddons />', () => {
+  const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
+
+  function render({ ...props }) {
+    return shallowRender(
+      <FeaturedAddonsBase i18n={getFakeI18nInst()} {...props} />
```

Comment:
If you did `<FeaturedAddons store={store} ...>` (i.e. use the default component, not the base) won't that invoke `mapStateToProps` automatically?